app/assets/stylesheets/to_change_everything/common/_font_styling.scss

```diff
@@ -0,0 +1,223 @@
+  @media (min-width:1671px) { @include imagetext(4.5vw, 4.25vw); }
+}
+
+.right p {
+  @include body-text(14px, 20px);
+  text-indent: 20px;
+
+  @media (min-device-width:200px) and (max-width:600px) { @include body-text(14px, 20px); }
+  @media (min-width:601px) and (max-width:1023px) {
+    @include body-text(18px, 26px);
+    text-indent: 30px; }
+  @media (min-width:1024px) and (max-width:1200px) {
+    @include body-text(18px, 26px);
+    text-indent: 30px; }
+  @media (min-width:1201px) and (max-width:1670px) {
+    @include body-text(1vw, 1.6vw);
+    text-indent: 1.3vw; }
+  @media (min-width:1671px) {
+    @include body-text(.88vw, 1.5vw);
+    text-indent: 1.3vw; }
+}
+
+.rightintro h3 {
+  @include intro-header(15vw, 14vw);
+  text-shadow: $black-text-shadow;
+  text-align: right;
+  text-indent: 4vw;
+  color: white;
+  letter-spacing: -.03em;
+
+  @media (min-device-width:200px) and (max-width:600px) { @include intro-header(15vw, 14vw); }
+  @media (min-width:601px) and (max-width:1023px) { @include intro-header(15vw, 14vw); }
+  @media (min-width:1024px) and (max-width:1200px) { @include intro-header(6vw, 5.25vw); }
+  @media (min-width:1201px) and (max-width:1670px) { @include intro-header(6vw, 5.25vw); }
+  @media (min-width:1671px) { @include intro-header(6vw, 5.25vw); }
+}
+
+.leftintro p {
+  @include intro-text(4vw, 5.5vw);
+  text-indent: 4vw;
+
+  @media (min-device-width:200px) and (max-width:600px) { @include intro-text(4vw, 5.5vw); }
+  @media (min-width:601px) and (max-width:1023px) { @include intro-text(4vw, 5.5vw); }
+  @media (min-width:1024px) and (max-width:1200px) { @include intro-text(4vw, 5.5vw); }
+  @media (min-width:1201px) and (max-width:1670px) {
+    @include intro-text(1.55vw, 2vw);
+    text-indent: 2vw; }
+  @media (min-width:1671px) {
+    @include intro-text(1.55vw, 2vw);
+    text-indent: 2vw; }
+}
+
+#outro h3 {
+    color: black;
+    text-shadow: none;
+    @media (min-width:1201px) { color: white; }
+}
+
+#outro .leftintro p {
+  @include outro-text(4.5vw, 6vw);
+  text-indent: 0vw;
+  color: white;
+
+  @media (min-device-width:200px) and (max-width:600px) { @include outro-text(4.5vw, 6vw); }
+  @media (min-width:601px) and (max-width:1023px) { @include outro-text(4.5vw, 6vw); }
+  @media (min-width:1024px) and (max-width:1200px) { @include outro-text(4.5vw, 6vw); }
+  @media (min-width:1201px) and (max-width:1670px) {
+      @include outro-text(1.7vw, 2.3vw);
+      color: black; }
+  @media (min-width:1671px) {
+      @include outro-text(1.7vw, 2.3vw);
+      color: black; }
+}
+
+#anarchy p {
+  @include anarchy-text(16px, 24px);
+  text-indent: 22px;
+  color: white;
+  -webkit-font-smoothing: antialiased;
+  -moz-font-smoothing: antialiased;
+  -o-font-smoothing: antialiased;
+
+  @media (min-device-width:200px) and (max-width:600px) { @include anarchy-text(16px, 24px); }
+  @media (min-width:601px) and (max-width:1023px) {
+    @include anarchy-text(20px, 32px);
+    line-height: 32px; }
```

Comment:
I don't love this style with the closing `}` on the same line. It's shorter, yes. But the trade off is that editors can't easily auto-indent to reveal syntax / nesting issues.

Reply:
fixed